### DISCUSSION TITLE
特定のキーが反応しなかったり連打されたりする

### DISCUSSION THREAD

**Discussion by KIUPICHI** [Yowkees/keyball#431] [Q&A] [answered]
![S__13991938](https://github.com/Yowkees/keyball/assets/140035694/434e8f0b-3271-44d9-872c-25d852363b1b)
お世話になっております。
画像の◯で囲んだキーが、押す際に反応しなかったり、1度押しただけで2回入力されたりします。
（10回押下すると1回程度無反応か、2回入力したりする程度の頻度です）

USBの左右を入れ替えたり、TRRSケーブルを交換しても症状が出ます。
改善方法がありましたらご教授いただけると幸いです。

**Comment by apio1969** [comment, marked as answer]
こんにちは～
私も以前同じような症状になったことがあるので,参考になればと思いますが、
私の場合は,特定のキーが２回連続で入力される現象でした。
連続で入力されるキーのキースイッチ接続部分をハンダごてで温め直し、さらに少しハンダを追加することで
症状がでなくなりました。
キーソケットのハンダ付けする部分の形状が昔のと変わっていることもあるので、ソケットのハンダ部分の接続不良
ではないかと思います。
一度お試しを。　改善されることを願っています。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。また、apio様もアドバイスありがとうございます。

apio様のアドバイスの通り、該当するキーソケットのはんだ付け部分が原因となる場合が多いため、再度はんだ付けの見直しをお願いいたします。
キーソケットに対応するダイオードは添付しました写真の色どうしのものになりますので、こちらのはんだ付けも併せてご確認いただけますと幸いです。
![287105715-434e8f0b-3271-44d9-872c-25d852363b1b](https://github.com/Yowkees/keyball/assets/143698349/ea3a4b45-628d-4bff-a07a-360316f88ecc)

**Comment by KIUPICHI** [comment]
アドバイスに従ってはんだ付けを見直してみたところ、正常に反応するようになりました！
大変気に入っているキットなので解消してよかったです。お二人共ありがとうございました！

**Reply by Ryo624** [reply]
無事に解決していただきありがとうございます！
ぜひ末永くご愛用していただければ幸いです。
